arctic/store/_ndarray_store.py

```diff
@@ -129,9 +126,12 @@ def can_delete(self, version, symbol):
     def can_read(self, version, symbol):
         return version['type'] == self.TYPE
 
-    def can_write(self, version, symbol, data):
+    def can_write(self, version, symbol, data, **kwargs):
         return isinstance(data, np.ndarray) and not data.dtype.hasobject
 
+    def can_update(self, version, symbol, data, **kwargs):
```

Comment:
NdarrayStore is the base class. I can just remove it from here (its covered in the PandasStore class), but that could cause options in some rare situations. Are you saying you'd want an even "baser" class? Something like "VersionedObjectStore" or something like that that everything here and ahl.mongo can inherit from?